### DISCUSSION TITLE
d

### DISCUSSION THREAD

**Discussion by ditofirrizqi** [dewanakl/undangan#11] [General]
bang cara ngehapus riwayat komentar gimana ya

**Comment by dewanakl** [comment]
Untuk menghapus riwayat chat belum ada mas, tapi kalau chat individu masing masing nanti bakal ada selama belum clear cache pada browsernya.

Karena untuk menghapus riwayat komentar saya asumsikan temen temen semuanya memakai server pribadi jadi bisa dengan mudah mengelolanya.

Terimakasih
